DemoApp/DemoApp/Views/TMCardPage.xaml

```diff
@@ -129,8 +129,7 @@
           </StackLayout>
         </StackLayout>
       </Modus:TMCard>
-      <Modus:TMCard x:Name="card4"
-                    WidthRequest="250">
+      <Modus:TMCard x:Name="card4">

```

Comment:
Can you add padding property in demo app card view page. So that we can change dynamically

Reply:
I wouldn't stress about that. We don't want to show that as a feature of the card. I don't think of a scenario where the user will set a padding other than the default one and 0.